ui/src/components/app_template.vue

```diff
@@ -50,6 +50,12 @@
   import the_footer from 'components/the_footer'
   import uploads_progress from 'components/uploads_progress'
 
+  has_files = (evt) ->
+    if evt.dataTransfer.types
+      for type in evt.dataTransfer.types
+        if type == 'Files' then return true
+    return false
```

Comment:
`yarn add lodash.some`

`has_files = (evt) -> some evt.dataTransfer.types, t -> t.type is 'Files'`